BLSQ/iaso
# POLIO-1284: Add Scopes download in bulk button and date range selection - backend

Explain what problem this PR is resolving
- Add export csv for all round scopes in the calendar page
Related JIRA tickets : [POLIO-1284](https://bluesquare.atlassian.net/browse/POLIO-1284)

## Self proofreading checklist


## Changes
- Add export to csv button
- Launch a dialog to select start_from and start_to date
- Add on the campains a method to export csv file containing all rounds scopes

## How to test
- Go on the calendar polio page
- At the botton right click on the export csv button
- Choose start from and start to dates
- Confirm
- Check the exported csv file

## Print screen / video
[Screencast from 2023-11-15 13-50-39.webm](https://github.com/BLSQ/iaso/assets/19631540/bc9a3186-7da4-4037-824b-89160dfe0f15)



[POLIO-1284]: https://bluesquare.atlassian.net/browse/POLIO-1284?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
